contracts/land/LANDRegistry.sol

```diff
@@ -479,4 +472,87 @@ contract LANDRegistry is Storage, Ownable, FullAssetRegistry, ILANDRegistry {
     assembly { size := extcodesize(addr) }
     return size > 0;
   }
+
+  //
+  // LAND Ping
+  //
+
+  /**
+   * @dev Set date from when the LAND Ping feature should be enabled
+   * @param _gracePeriod Desired amount of time in miliseconds to enable feature
+   */
+  function setGracePeriod(uint _gracePeriod) external onlyDeployer {
+    gracePeriod = _gracePeriod;
+  }
+
+  /**
+   * @dev Set amount of time that should pass for a LAND to be transferred to
+   * a new onwer
+   * @param _deemPeriod Desired amount of time in miliseconds for a LAND to decay
+   */
+  function setDeemPeriod(uint _deemPeriod) external onlyDeployer {
+    deemPeriod = _deemPeriod;
```

Comment:
same as above @eordano @abarmat maybe this one has more sense. The grace period may change due the time could take to develop the auction feature, but changing this one could be polemic

Reply:
Use `uint256`

Reply:
Can we emit an event here too?